pythonforandroid/bootstraps/common/build/jni/application/src/start.c

```diff
@@ -243,9 +275,71 @@ int main(int argc, char *argv[]) {
 #if PY_MAJOR_VERSION < 3
   PyRun_SimpleString("import site; print site.getsitepackages()\n");
 #endif
-
   LOGP("AND: Ran string");
 
+#if PY_MAJOR_VERSION >= 3
+  /* Workaround crash in ctypes.find_library() which is caused by its
+   * internal use of /bin/sh, and looking in the wrong folders on Android.
+   *
+   * The code needs 'env_argument' as baked path, so we need to insert this:
+   */
+  LOGP("ctypes workaround insert");
+  char code_with_placeholder[] = (""
```

Comment:
Would it make sense and be easy enough to have this large string loaded from a file instead? That way it would make it easier to edit in case it's needed?

Reply:
The `clean` way of how to do that would I think be to install it into the site-packages, but that might be a little complex to look at. (Like, harder to figure out for people who try to understand how the boot process works) The other alternative I can think of is adding the file to the asset folder, and then directly `open()`ing it in that inline python and using `exec()` or something.

I'm a bit undecided on how much 'better' these two solutions are. They would certainly be easier to edit, but I'm not sure if it wouldn't also be more confusing about what is going on at boot.

Reply:
Basically I can't think of the most obvious mechanism and it would introduce additional complexity, so I'm a bit torn

Reply:
OK thanks :+1: